app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -158,6 +180,186 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  fun testStateFragment_clickPlayExploration_explorationLoadsSuccessfully() {
+    ActivityScenario.launch(HomeActivity::class.java).use {
+      onView(withId(R.id.play_exploration_button)).perform(click())
+      intended(hasComponent(ExplorationActivity::class.java.name))
+      onView(atPosition(R.id.state_recycler_view, 0)).check(matches(hasDescendant(withId(R.id.interaction_button))))
+    }
+  }
+
+  @Test
+  fun testStateFragment_loadExplorationTest5_submitButtonIsDisplayed() {
```

Comment:
How is this test different than the one above? Is it meant to click a different button or type something in?

Reply:
Removed this test-case because `testStateFragment_clickPlayExploration_explorationLoadsSuccessfully` already covers this test-case.